app/src/sharedTest/java/org/oppia/app/parser/HtmlParserTest.kt

```diff
@@ -112,6 +110,19 @@ class HtmlParserTest {
     onView(withId(R.id.test_html_content_text_view)).check(matches(withText(textView.text.toString())))
   }
 
+  @Test
+  fun testHtmlContent_handleHtmlListTags_parsedHtmlDisplaysStyledText() {
+    val textView = activityTestRule.activity.findViewById(R.id.test_html_content_with_ordered_list_text_view) as TextView
+    val htmlParser = htmlParserFactory.create(/* entityType= */ "", /* entityId= */ "", /* imageCenterAlign= */ true)
+    val htmlResult: Spannable = htmlParser.parseOppiaHtml(
+      getResources().getString(R.string.faq_answer_1),
+      textView
+    )
+    assertThat(textView.text.toString()).isEqualTo(htmlResult.toString())
```

Comment:
This seems to be testing that the test activity is setup correctly rather than ensuring the parsing logic is correct (e.g. if you switch out the FAQ string with something that doesn't have an ordered list here & above, it would still pass).

One option might be to obtain the Spannable from the TextView and ensure the correct spans are present to render an ordered list, and any other string changes added to support the indentation.

Reply:
Added testcases showing spans.